### PR TITLE
Marcio/web-gateway

### PR DESCRIPTION
**Pull request by marciocloudflare**
added pcx metadada
corrected links

### CODE REVIEW

**Comment by marciocloudflare** [comment]
Hi @Lekensteyn @lukevalenta @thibmeu. Please ignore my previous PR as I made a mistake and created it from another branch I was working for another project. This PR is to add PCX metadata and I've gone and corrected some links that were broken. This is meant only for our team's reporting tools and doesn't affect anything on the presentation side of the public docs

**Review by deadlypants1973 — APPROVED** [review]

File: products/distributed-web/src/content/ethereum-gateway/about-eth.md [hunk 1]
```diff
@@ -57,13 +58,13 @@ sent to the wider network and added to the consensus.
...
 Cloudflare's Gateway. To learn more about how to do this see [Interacting with
-the Ethereum network](./interacting-with-the-eth-gateway).
+the Ethereum network](../interacting-with-the-eth-gateway).
 
 ## Connect your website to the gateway
 
 If you want to be able to access the Ethereum network accessible from a custom
 domain name, you can do that using Cloudflare’s Ethereum Gateway. To
-learn how, check out [Connecting your Website](./connecting-your-website).
+learn how, check out [Connecting your Website](../connecting-your-website).
```
Comment (deadlypants1973) [inline products/distributed-web/src/content/ethereum-gateway/about-eth.md]:
```suggestion
learn how, refer to [Connecting your Website](../connecting-your-website).
```

Comment (deadlypants1973) [reply products/distributed-web/src/content/ethereum-gateway/about-eth.md]: This wording was suggested to me by Claire. It's a bit more technical writing-esque than 'check out' if you want to use it.

File: products/distributed-web/src/content/ipfs-gateway/connecting-website.md [hunk 2]
```diff
@@ -120,7 +121,7 @@ If your website is on Cloudflare, the DNS settings are accessible from your
 dashboard and can be managed through [our
 API](https://api.cloudflare.com/#dns-records-for-a-zone-update-dns-record). If
 your website is not on Cloudflare, and you need help finding the DNS records,
-[look here](https://support.google.com/projectshield/answer/6358886?hl=en/).
+[look here](https://support.projectshield.withgoogle.com/s/article/Find-your-Domain-Name-System-DNS-records?language=en_US).
```
Comment (deadlypants1973) [inline products/distributed-web/src/content/ipfs-gateway/connecting-website.md]:
```suggestion
[refer to Project Shield's guide.](https://support.projectshield.withgoogle.com/s/article/Find-your-Domain-Name-System-DNS-records?language=en_US).
```

**Pull request merged**